CONTRIBUTING.md

```diff
@@ -33,91 +33,108 @@ These are one time installations required to be able to test your changes locall
 
 1. Install [Node.js](https://nodejs.org/en/download/) for your platform
 1. Install [VS Code](https://code.visualstudio.com/download) for your platform
-1. Install the dependencies. From the repository root run:
+1. Install the dependencies. From the repository root (`~/vscode`) run:
```

Comment:
The existing instructions in the file don't assume the contributor is following the setup in https://github.com/github/vscode-github-actions/blob/main/docs/development.md. (npm workspaces for changes across several packages). They are focused on changes to the vscode-github-actions repository.

If users are cloning just this one repo, repository root is likely (but not neccessarily) called `vscode-github-actions` and we cannot know what directory path they cloned this into